### DISCUSSION TITLE
Woche 9

### DISCUSSION THREAD

**Discussion by Christian-2003** [Christian-2003/CrInGE#36] [Student Exchange]
Hallo zusammen,

diese Woche haben wir uns hauptsächlich mit der Fertigstellung des :page_facing_up: **ASR** mittels dem arc42-Template befasst.

Erfreulicherweise konnten wir alle Kapitel bearbeiten, damit wir uns in der nächsten Woche vollends auf die :computer: **Implementierung** und :loudspeaker: **Abschlusspräsentation** konzentrieren können.

Das vollständige ASR könnt ihr [hier](https://github.com/Christian-2003/CrInGE/blob/master/docs/ArchitektonischRelevanteAnforderungen.md) finden.

###### Stand der Implementierung
Neben dem ASR haben wir selbstverständlich auch weiter an unserer :cd: **Software** programmiert. Diese Woche konnten wir endlich das :movie_camera: **Rendering** durch die Engine fertigstellen.

Damit ihr euch das visuell besser vorstellen könnt, haben wir hier ein kleines Beispiel für euch:

![image](https://github.com/Christian-2003/CrInGE/assets/80711177/32a350cf-ab33-4b88-98a5-0513ea03ae17)

Das wäre dann auch schon alles für diese Woche. Im nächsten Blog werden wir euch weiter über das Projekt informieren.

**Comment by 999Marlon** [comment]
Sieht sehr sehr gut und vorallem ausführlich aus. Ich habe nichts zu kritisieren, sehr stark
